lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html

```diff
@@ -299,6 +307,24 @@
   stampTemplate('unoptimized-images-tmpl', document.body);
 }
 
+function deprecationsTest() {
+  stampTemplate('deprecations-tmpl', document.head);
+
+  // Some APIs atm are not triggering Chrome's deprecation messags. https://crbug.com/680832.
```

Comment:
`messages`, but maybe something more like `// TODO: some deprecated API messages are not currently being logged correctly. See: https://crbug.com/680832`